contentcuration/contentcuration/frontend/channelEdit/views/sync/SyncResourcesModal.vue

```diff
@@ -228,4 +265,10 @@
     white-space: unset;
   }
 
```

Comment:
For the `.v-list_tile` styling on line 258, I can't attach a comment directly to that but could you replace with the following:
```css
  /deep/ .v-list__tile {
    align-items: flex-start;
    height: unset;
    padding: 8px 0;
  }
```

the existing styles don't actually do anything.